### PR TITLE
feat(@vtmn/css): add toast component

### PR DESCRIPTION
**Pull request by GaspardMathon**
## Pull Request checklist

🚨 Please review the [guidelines for contributing](../blob/main/CONTRIBUTING.md) to this repository.

- [x] Make sure you are requesting to **pull a topic/feature/bugfix branch**. Don't request your main!
- [x] Check the commit's or even all commits' message styles matches our requested structure.
- [x] Check your code additions will fail neither code linting checks nor unit test.
- [x] Check PR name, it must follow the https://www.conventionalcommits.org pattern
- [x] If it's a new component in CSS, ask for design review

## Description

- Add toast component

## Does this introduce a breaking change?

- No

❤️ Thank you!

### CODE REVIEW

**Comment by xavhan** [comment]
I removed the frontend-community review as PR is not ready yet

File: packages/sources/css/src/components/toast/src/index.css [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  font-family: var(--vtmn-typo_font-family);
+  font-size: var(--vtmn-typo_text-2-font-size);
+  line-height: var(--vtmn-typo_text-2-line-height);
+  color: var(--vtmn-color_white);
+  background-color: var(--vtmn-color_black);
+  box-shadow: var(--vtmn-shadow_200);
+  border-radius: rem(4px);
+}
+
+.vtmn-toast > p {
```
Comment (xavhan) [inline packages/sources/css/src/components/toast/src/index.css:23]: any particular reason to force the content to be in a p ?

File: packages/showcases/css/stories/components/toast/examples/overview.html [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  <div class="vtmn-toast">
+    <p>
+      This is the description of a snackbar This is the description of a
+      snackbar
+    </p>
+  </div>
+</div>
+
+<div class="block">
+  <div class="vtmn-toast vtmn-toast_icon-info">
```
Comment (lauthieb) [inline packages/showcases/css/stories/components/toast/examples/overview.html:11]:
```suggestion
  <div class="vtmn-toast vtmn-toast_with-icon-info">
```

File: packages/showcases/css/stories/components/toast/examples/overview.html [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+  <div class="vtmn-toast vtmn-toast_icon-close">
+    <p>
+      This is the description of a snackbar This is the description of a
+      snackbar
+    </p>
+  </div>
+</div>
+
+<div class="block">
+  <div class="vtmn-toast vtmn-toast_icon-info vtmn-toast_icon-close">
```
Comment (lauthieb) [inline packages/showcases/css/stories/components/toast/examples/overview.html:29]:
```suggestion
  <div class="vtmn-toast vtmn-toast_with-icon-info vtmn-toast_icon-close">
```

File: packages/sources/css/src/components/toast/src/index.css [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+  overflow: hidden;
+  white-space: nowrap;
+  text-overflow: ellipsis;
+}
+
+.vtmn-toast_icon-info > p::before {
```
Comment (lauthieb) [inline packages/sources/css/src/components/toast/src/index.css:29]:
```suggestion
.vtmn-toast_with-icon-info > p::before {
```

**Review by lauthieb — CHANGES_REQUESTED** [review]

**Review by lauthieb — APPROVED** [review]
GREAT JOB :tada:

https://media1.giphy.com/media/3o6Ei00MRU2WnGpcGI/giphy.gif?cid=5a38a5a2lm2x5z6fmlsughl2s177e83lx01c4isg8ixihtqt&rid=giphy.gif&ct=g

**Pull request merged**